### PR TITLE
make htpasswd tasks remote actions instead of local

### PR DESCRIPTION
**Pull request by ewolinetz**
to address #3921 

cc @mwoodson

### CODE REVIEW

**Comment by sdodson** [comment]
Currently we install httpd-tools via the rpm dependencies for openshift-ansible[1]. We could change this[2] so that it's not conditional based on the IDP in use but we still wouldn't have htpasswd on atomic host. How should this be handled on atomic host?

1 - https://github.com/openshift/openshift-ansible/blob/master/openshift-ansible.spec#L26
2 - https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master/tasks/main.yml#L74-L78

**Comment by sdodson** [comment]
https://github.com/openshift/openshift-ansible/issues/1071 related, discusses some solutions for htpasswd on atomic host

File: roles/openshift_metrics/tasks/install_support.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 - name: Check control node to see if htpasswd is installed
```
Comment (jcantrill) [inline roles/openshift_metrics/tasks/install_support.yaml]: Change name

**Comment by ewolinetz** [comment]
Updated logic to use docker run instead for generating a htpasswd file @sdodson

**Comment by ewolinetz** [comment]
aos-ci-test

**Pull request merged**